src/Stratis.Features.Collateral/JoinFederationRequestMonitor.cs

```diff
@@ -66,11 +66,12 @@ private void OnBlockConnected(BlockConnected blockConnectedData)
                         continue;
 
                     // Check if the collateral amount is valid.
-                    // Voting only works on non-multisig members.
                     decimal collateralAmount = request.CollateralAmount.ToDecimal(MoneyUnit.BTC);
-                    if (collateralAmount != CollateralPoAMiner.MinerCollateralAmount)
+                    var expectedCollateralAmount = ((PoANetwork)this.network).StraxMiningMultisigMembers.Any(m => m == request.PubKey)
```

Comment:
I see you are doing this in a few places... Can yo move to a static method somewhere?

Reply:
Done.